creator-node/src/routes/tracks.js

```diff
@@ -421,6 +421,19 @@ router.post(
           throw new Error('Cannot create track without transcodedTrackUUID.')
         }
 
+        // Verify that blockchain track id is owned by user attempting to upload it
+        const serviceRegistry = req.app.get('serviceRegistry')
+        const { libs } = serviceRegistry
+        const trackResp = await libs.contracts.TrackFactory.getTrack(
+          blockchainTrackId
+        )
+        // eslint-disable-next-line eqeqeq
```

Comment:
is this a real eslint one-liner LOL

Reply:
yep it is https://eslint.org/docs/latest/rules/eqeqeq. but the real question is why do we need it?

Reply:
I see a similar SequelizeUniqueConstraintError logged for the AudiusUsers table, so adding another check should prevent that as well. I just pushed this out now. also, see my comment below for why this is using eqeq instead of eqeqeq